Kotlin asinh (#3622)
Codecademy/docs: content/kotlin/concepts/math-methods/terms/asinh/asinh.md

---
Title: 'asinh()'

> Description: 'Returns the inverse hyperbolic sine of a numeric argument'

Reviewer: ```suggestion
Description: 'Returns the inverse hyperbolic sine of a numeric argument.'
```

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Double'
  - 'Float'
  - 'Methods'
  - 'Functions'
CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---

The **`asinh()`** function returns the inverse hyperbolic sine of a number.

## Syntax

```pseudo
asinh(x)
```

The function accepts one mandatory parameter `x`. It can be any value i.e. `negative`, `positive`, or `zero`. The parameter can be of  `double`, `float` or `long` datatype. 

The function returns the inverse hyperbolic sine of the argument. The return value can be of `double`, `float` or `long` datatype.

## Example

> 

Reviewer: ```suggestion
```
Delete extra blank line.

Author: Kindly check. I have fixed it, sir !

> In this example below we are trying to find the value of inverse hyperbolic sine of 1.0

Reviewer: ```suggestion
The example below is finding the value of the inverse hyperbolic sine of 1.0
```
